frontend/src/components/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -32,13 +34,29 @@ type ReviewRow = {
 
 const AdminDashboard = (): React.ReactElement => {
   const [data, setData] = useState<ReviewResponse[]>([]);
+  const { notifications } = useContext(NotificationContext);
+  const toast = useToast();
 
   useEffect(() => {
     reviewAPIClient.getReviews().then((allReviews: ReviewResponse[]) => {
       setData(allReviews);
     });
   }, []);
 
+  useEffect(() => {
+    if (notifications.includes("published")) {
```

Comment:
instead of checking `includes` - can we check if it's some exact ID or enum type?

Reply:
Intentionally used `includes` so that the notifications array can support other kinds of "global" notifications without the need for a new context